src/app/public/modules/phone-field/phone-field-input.directive.ts

```diff
@@ -87,8 +87,8 @@ export class SkyPhoneFieldInputDirective implements OnInit, OnDestroy, AfterView
     if (value) {
       let formattedValue = this.formatNumber(value.toString());
 
-      if (this.phoneFieldComponent.selectedCountry.iso2 !== this.phoneFieldComponent.defaultCountry) {
-        formattedValue = '+' + this.phoneFieldComponent.selectedCountry.dialCode +
+      if (this.phoneFieldComponent.selectedCountry.iso2 !== this.phoneFieldComponent.defaultCountry && !value.startsWith('+')) {
```

Comment:
This condition's readability could be improved by creating a few boolean variables that would help a future contributor parse out what's going on.

Maybe something like:
```
const hasDialCode = (value.startsWith('+'));
const isDefaultCountrySelected = (
  this.phoneFieldComponent.selectedCountry.iso2 ===
  this.phoneFieldComponent.defaultCountry
);

if (!isDefaultCountrySelected && !hasDialCode) {
  ...
}
```